packages/devtools_app/lib/src/debugger/file_search.dart

```diff
@@ -91,20 +112,20 @@ class _FileSearchFieldState extends State<FileSearchField>
 
   void _onSelection(String scriptUri) {
     final scriptRef = _scriptsCache[scriptUri];
-    widget.controller.showScriptLocation(ScriptLocation(scriptRef));
+    widget.debuggerController.showScriptLocation(ScriptLocation(scriptRef));
     _onClose();
   }
 
   void _onClose() {
-    _autoCompleteController.closeAutoCompleteOverlay();
-    widget.controller.toggleFileOpenerVisibility(false);
+    widget.autoCompleteController.closeAutoCompleteOverlay();
+    widget.debuggerController.toggleFileOpenerVisibility(false);
     _scriptsCache.clear();
   }
 
   @override
   void dispose() {
     _onClose();
-    _autoCompleteController.dispose();
+    widget.autoCompleteController.dispose();
```

Comment:
this shouldn't be disposed here. we don't own the _autoCompleteController. The state object that created it and passed it in should own it.